setup.sh

```diff
@@ -100,58 +111,94 @@ check() {
         exit 1
     fi
 
+    # setup environment file
     if [ ! -f "_env" ]; then
-        echo "Creating a configuration file..."
-        cp _env.example _env
+        echo '# Environment variables for for WordPress site' > _env
+        echo '# please include the scheme http:// or https:// in the URL variable' >> _env
+
+        echo 'WORDPRESS_URL=http://'wordpress.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.triton.zone >> _env
```

Comment:
yeah, i'm working on some updates and testing right now. Going to make your requested changes to the readme, I'll include this in there too.